freshworks.com.domain-authentication.json

```diff
@@ -0,0 +1,76 @@
+{
+  "providerId": "freshworks.com",
+  "providerName": "Freshworks",
+  "serviceId": "domain-authentication",
+  "serviceName": "Freshworks",
+  "logoUrl": "https://dam.freshworks.com/m/1d230ee78c07681a/original/headerLogoLight.webp",
+  "version": 1,
+  "syncPubKeyDomain": "freshworks.com",
+"variableDescription": "Self explanatory in the variable names",
+"description": "CNAME records are utilized for domain verification, enabling Freshworks to authenticate emails sent on behalf of customers. Additionally, DKIM records are employed to further authenticate these emails.",
+  "records": [
+    {
+      "groupId": "dkim1",
+      "host": "%cname_host%",
+      "type": "CNAME",
+      "pointsTo": "%cname_pointsTo%",
+      "ttl": 60
```

Comment:
This is a very short TTL. Not sure, but some DNS providers might no be super happy about it.

Reply:
Increased the TTL